testing/sdk_install.py

```diff
@@ -221,6 +208,37 @@ def uninstall(
             shakedown.pretty_duration(finish - cleanup_start),
             shakedown.pretty_duration(finish - start)))
 
+    state_summary = sdk_cmd.cluster_request('GET', '/mesos/state-summary').json()
+
+    # Sanity check: There should be no dangling resources in the state summary (DCOS-30314)
+    found_dangling_resources = False
+    service_role = '{}-role'.format(service_name)
```

Comment:
This doesn't work for foldered service names.

Reply:
`sdk_install` was already doing this correctly elsewhere, "upstreamed" that into `sdk_utils` and switched users to calling that.